docs/tutorials/get-started-with-ngo.md

```diff
@@ -0,0 +1,619 @@
+---
+id: get-started-ngo
+title: Get started with NGO
+---
+
+Use this guide to learn how to create your first NGO project. It walks you through creating a simple Hello World project that implements the basic features of Netcode for GameObjects (NGO).
+
+## Prerequisites
+
+Before you start, ensure you have the following prerequisites:
+
+- An active Unity account with a valid license.
+- The [Unity Hub](https://unity.com/download).
+- A NGO-supported version of the Unity Editor. See [NGO requirements](../installation/installation.md).
+
+Before continuing, create a new project using Unity Editor version 2021.3 or later.
+
+## Install Netcode for GameObjects
```

Comment:
This content is already covered in https://docs-multiplayer.unity3d.com/netcode/current/installation/install, which is already linked above. I think we could remove this section or just redirect to that other page here as well. Also, I suppose we should also replace the links from the installation page to the Hello World and Golden Path tutorials with links towards this new version?

Reply:
Oh, good shout. Yeah, I'll need to update those links.